lms/djangoapps/grades/management/commands/recalculate_subsection_grades.py

```diff
@@ -0,0 +1,92 @@
+"""
+Command to compute all grades for specified courses.
+"""
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from datetime import datetime
+import logging
+from pytz import utc
+
+from django.core.management.base import BaseCommand
+from lms.djangoapps.grades.constants import ScoreDatabaseTableEnum
+from lms.djangoapps.grades.tasks import recalculate_subsection_grade_v3
+from courseware.models import StudentModule
+from student.models import user_by_anonymous_id
+from submissions.models import Submission
+from track.event_transaction_utils import create_new_event_transaction_id, set_event_transaction_type
+from util.date_utils import to_timestamp
+
+log = logging.getLogger(__name__)
+
+DATE_FORMAT = "%Y-%m-%d %H:%M"
+
+PROBLEM_SUBMITTED_EVENT_TYPE = u'edx.grades.problem.submitted'
```

Comment:
This can be imported from here: https://github.com/edx/edx-platform/blob/master/lms/djangoapps/grades/signals/handlers.py#L38